test/sprockets/bumble_d/transformer_test.rb

```diff
@@ -8,14 +8,7 @@ module Sprockets
   module BumbleD
     class TransformerTest < Minitest::Test
       def test_it_compiles_es6_features_to_es5
-        input = {
-          content_type: 'application/ecmascript-6',
-          data: 'const square = (n) => n * n',
-          metadata: {},
-          load_path: File.expand_path('../foo', __FILE__),
-          filename: File.expand_path('../foo/bar.es6', __FILE__),
-          cache: Sprockets::Cache.new
-        }
+        input = input_exemplar
```

Comment:
I've addressed this by splitting out commit 6b806131507ff805947a12a4b75dc16b322e4bf7